### PR TITLE
V4.2.x

### PR DESCRIPTION
**Pull request by jungwire**
* Runs with HTTP
* Is installable (But needs some commands executed in the cli) should be noted as TODO for further development (so that no cli is necessary)
* Was tested by PN

### CODE REVIEW

**Comment by Maexchen** [comment]
Local Version Check => Tested
Bibbox Version Check => Tested

@README.md
Install within BIBBOX => Highlight that future steps are needed 

@README.md
Install within LOCAL => Highlight that future steps are needed 


@INSTALL-APP.md 
Some Command Line Work
add example for <instance_url>

Typo in Headline "Retart ElabFTW"

File: INSTALL-APP.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+`[services.d] done.`
+
+Once installed we have to do some command line work to initilize elabftw and make it comaptible with HTTP instead of HTTPS. 
+Log into our bibbox server (e.g. via ssh) and execute the following commands:
+
+* Import the database structure:
+  * `docker exec -it <instance_name>-web bin/install start`
+* Disable secrue cookies (Incompatible with HTTP):
+  * `docker exec -it <instance_name>-web sed -i 's/session.cookie_secure = true/session.cookie_secure = false/' ../etc/php8/php.ini`
+* Set the app url in database (including the http://):
```
Comment (Maexchen) [inline INSTALL-APP.md:15]: add example for <instance_url>

File: INSTALL-APP.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  * `docker exec -it <instance_name>-mysql mysql -u elabftw --database="elabftw" --password="<password>" --execute='UPDATE config SET conf_value = "http://<instance_url>:80" WHERE conf_name = "url";'`
+
+### Retart ElabFTW
```
Comment (Maexchen) [inline INSTALL-APP.md]: Typo in Headline "Retart ElabFTW"

File: INSTALL-APP.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Once installed we have to do some command line work to initilize elabftw and make it comaptible with HTTP instead of HTTPS. 
+Log into our bibbox server (e.g. via ssh) and execute the following commands:
+
+* Import the database structure:
+  * `docker exec -it <instance_name>-web bin/install start`
+* Disable secrue cookies (Incompatible with HTTP):
+  * `docker exec -it <instance_name>-web sed -i 's/session.cookie_secure = true/session.cookie_secure = false/' ../etc/php8/php.ini`
+* Set the app url in database (including the http://):
+  * `docker exec -it <instance_name>-mysql mysql -u elabftw --database="elabftw" --password="<password>" --execute='UPDATE config SET conf_value = "http://<instance_url>:80" WHERE conf_name = "url";'`
```
Comment (Maexchen) [inline INSTALL-APP.md:16]: add example for <instance_url>

File: README.md [hunk 4]
```diff
@@ -1,2 +1,53 @@
...
+ 
+The default values for the standalone installation are:
+
+ * DB_PASSWORD = password
+
+## Docker Images Used
+ * [elabftw/elabimg:4.2.4](https://hub.docker.com/r/elabftw/elabimg), offical elabftw container 
+ * [mysql:8.0](https://hub.docker.com/_/mysql), offical mysql container
+ 
+## Standalone Installation
```
Comment (Maexchen) [inline README.md:30]: Install within BIBBOX => Highlight that future steps are needed

File: README.md [hunk 5]
```diff
@@ -1,2 +1,53 @@
...
+After the installation follow these [instructions](INSTALL-APP.md)
+
+## Hints
+* approx. time with medium fast internet connection: **10 minutes**
+* This is a modification of the offical ELabFTW (https://www.elabftw.net/) applictaion. It uses **HTTP** instead of ~~HTTPS~~. Please keep this in mind when using this app.
+* Admin User: **set after instalations**
+* MySQL password **set during installations**
+
+
+## Install within BIBBOX
```
Comment (Maexchen) [inline README.md:14]: Install within BIBBOX => Highlight that future steps are needed

**Review by Maexchen — CHANGES_REQUESTED** [review]
Local Version Check => Tested
Bibbox Version Check => Tested

File: INSTALL-APP.md (hunk 2 above)
Comment (jungwire) [reply INSTALL-APP.md]: Thank you --> fixed it

File: INSTALL-APP.md (hunk 3 above)
Comment (jungwire) [reply INSTALL-APP.md:16]: Example was added in 109f7804dac1cc1991074e133788a6c1712dde65

File: README.md (hunk 5 above)
Comment (jungwire) [reply README.md:14]: Further steps were expicitly mentioned in Standalone and BIBBOX install sections

**Review by Maexchen — APPROVED** [review]

**Pull request merged**